app-center-services/src/main/java/org/exoplatform/appcenter/entity/ApplicationEntity.java

```diff
@@ -32,7 +32,7 @@
     @NamedQuery(name = "ApplicationEntity.getAppByTitle", query = "SELECT app FROM ApplicationEntity app "
         + "WHERE app.title = :title"),
     @NamedQuery(name = "ApplicationEntity.getApplicationsByKeyword", query = "SELECT app FROM ApplicationEntity app "
-        + "WHERE app.title like :title OR app.description like :description OR app.url like :url"),
+        + "WHERE app.title like :title OR app.description like :description OR LOWER(app.url) like LOWER(:url) "),
```

Comment:
```suggestion
        + "WHERE app.title like :title OR app.description like :description OR LOWER(app.url) like :url "),
```
You can do the lowercase of `:url` in DAO instead to do it in SQL part.

Reply:
ok